x-pack/plugins/maps/public/connected_components/toolbar_overlay/_toolbar_overlay.scss

```diff
@@ -0,0 +1,68 @@
+.mapToolbarOverlay {
+  position: absolute;
+  top: ($euiSizeM + $euiSizeS) + ($euiSizeXL * 2); // Position and height of mapbox controls plus margin
+  left: $euiSizeM;
+  z-index: 2; // Sit on top of mapbox controls shadow
+}
+
+.mapToolbarOverlay__button,
+.mapToolbarOverlay__buttonGroup {
+  position: relative;
+  transition: transform $euiAnimSpeedNormal ease-in-out, background $euiAnimSpeedNormal ease-in-out;
+
+  @include kbnThemeStyle($theme: 'v7') {
+    // Overrides the .euiPanel default border
+    // sass-lint:disable-block no-important
+    border: none !important;
+
+    // Overrides the .euiPanel--hasShadow
+    &.euiPanel.euiPanel--hasShadow {
+      @include euiBottomShadowLarge;
+    }
+  }
+
+  &:hover {
+    transform: translateY(-1px);
+  }
+
+  // Removes the hover effect from the .euiButtonIcon because it would create a 1px bottom gap
+  // So we put this hover effect into the panel that wraps the button or buttons
+  .euiButtonIcon:hover {
+    transform: translateY(0);
+  }
+
+  // Removes the focus background state because it can induce users to think these buttons are "enabled".
+  // The buttons functionality are just applied once, so they shouldn't stay highlighted.
+  .euiButtonIcon:focus:not(:hover) {
+    background: none;
+  }
+}
+
+.mapToolbarOverlay__buttonGroup {
+  display: flex;
+  flex-direction: column;
+
+  .euiButtonIcon {
+    border-radius: 0;
+
+    @include kbnThemeStyle($theme: 'v7') {
+      &:first-child {
+        border-radius: $euiBorderRadius $euiBorderRadius 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0 $euiBorderRadius $euiBorderRadius;
+      }
+    }
+
+    @include kbnThemeStyle($theme: 'v8') {
+      &:first-child {
+        border-radius: ($euiBorderRadius * (2 / 3)) ($euiBorderRadius * (2 / 3)) 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0 ($euiBorderRadius * (2 / 3)) ($euiBorderRadius * (2 / 3));
+      }
+    }
```

Comment:
Same question here. Possible to get rid of these styles and instead put an `overflow: hidden;` on the parent element?